etcdctl/go.mod

```diff
@@ -1,6 +1,6 @@
 module go.etcd.io/etcd/etcdctl/v3
 
-go 1.23
+go 1.23.0
```

Comment:
Refer to PR description and https://github.com/etcd-io/etcd/pull/19477#issue-2876389476